packages/fxa-shared/metrics/amplitude-event.1.schema.json

```diff
@@ -54,18 +57,23 @@
       "pattern": "^[a-z]{1,}(-[a-zA-Z0-9]{1,})*$"
     },
     "country": {
+      "description": "The country in which the customer is located.",
       "type": "string"
     },
     "region": {
+      "description": "The province, state, or territory in which the customer is located.",
       "type": "string"
     },
     "os_name": {
+      "description": "The name of the operating system.",
       "type": "string"
     },
     "os_version": {
+      "description": "The version of the operating system.",
       "type": "string"
     },
     "device_model": {
+      "description": "The model number of the device.",
       "type": "string"
     },
     "event_properties": {
```

Comment:
Could we add descriptions for the `event_properties` and `user_properties` as well?

Reply:
These are legacy property names from when FxA used Amplitude, a third party library/business intelligence service. Their website might be a good place for this. In general, the `user_properties` are properties tied to the FxA user that don't vary by event, and `event_properties` are properties that are associated with a particular event and could vary event to event. That has been my observation anyway.